utils/helpers/get-select-knobs-config.js

```diff
@@ -0,0 +1,8 @@
+export default (items) => {
+  items = items.map((item) => item.name.replace('.', ''))
+  items.unshift(null)
```

Comment:
we don't need null value in select

there is issue in console
![image](https://user-images.githubusercontent.com/5370251/92906923-27632e00-f425-11ea-9101-df31a3584981.png)